academic_observatory_workflows/model.py

```diff
@@ -805,9 +817,16 @@ def make_papers(
             publisher_license=license_,
             publisher_is_free_to_read=publisher_is_free_to_read_,
             repositories=paper_repos,
+            in_scihub=bool(random.getrandbits(1)),
```

Comment:
What's the purpose of this randomness?

Reply:
Ah, just to make it so that 50% of the generated test papers are found in SciHub and so that 50% aren't